docs/docs/configuration/sessions.md

```diff
@@ -52,6 +52,31 @@ in redis via the `SETEX` command.
 Encrypting every session uniquely protects the refresh/access/id tokens stored in the session from
 disclosure.
 
+Additionally the browser only has to send a short Cookie with every request and not the whole JWT, which can get quite big.
+
+Two settings are used to configure the OAuth2-Proxy cookie lifetime:
+
+    --cookie-refresh duration   refresh the cookie after this duration; 0 to disable
+    --cookie-expire duration    expire timeframe for cookie     168h0m0s
+
+The "cookie-expire" value must be greater or equal to the Keycloak "client\_session\_idle" because if the cookie
```

Comment:
I don't think we want this to be keycloak specific since these docs are generic and apply to all providers equally

Reply:
You're right, I rephrased it.